server/api/user/user.controller.js

```diff
@@ -30,12 +30,35 @@ exports.stats = function(req, res) {
   // Only return users who are active and have a github login
   User.find({active: true, 'github.login': {$exists: true}}, '-salt -hashedPassword' ).exec(function (err, users) {
     if(err) return res.send(500, err);
+    var twoWeeks = new Date();
+    twoWeeks.setDate(twoWeeks.getDate()-14);
     var data = [];
-    for (var i = 0; i < users.length; i++){
-      data.push(users[i].stats);
+    var count = users.length;
+
+    var getCommits = function(user){
+      Commit.find()
+            .where('author.login').equals(String(user.github.login))
+            .where('date').gt(twoWeeks)
+            .exec(function(err, commits){
+                var commitList = [];
+                commits.forEach(function (c){
+                    commitList.push(c.toObject());
+                  }
+                )
```

Comment:
parenthesis should not go on next line- not really important but stylistically the parethesis is never really put on a new line